src/tlo/methods/schisto.py

```diff
@@ -241,63 +351,281 @@ def _load_parameters_from_workbook(self, workbook) -> dict:
+        params = self.parameters
+        coverage = params['mda_coverage']
+        target = params['mda_target_group']
+        frequency = params['mda_frequency_months']
+        districts = self.districts
+
+        # Create a new DataFrame with districts and schedule
+        prognosed_mda = pd.DataFrame(
+            index=pd.MultiIndex.from_product([districts, [frequency]], names=['District', 'Frequency_months']))
+
+        # Initialise columns with default values
+        prognosed_mda[['PSAC', 'SAC', 'Adults']] = 0
+
+        # Define default values for each column
+        default_values = {
+            'PSAC': 0,
+            'SAC': 0,
+            'Adults': 0
+        }
+
+        # Define the updates based on the target
+        updates = {
+            'PSAC_SAC': {
+                'PSAC': coverage,
+                'SAC': coverage,
+                'Adults': default_values['Adults']
+            },
+            'SAC': {
+                'SAC': coverage,
+                'PSAC': default_values['PSAC'],
+                'Adults': default_values['Adults']
+            },
+            'ALL': {
+                'PSAC': coverage,
+                'SAC': coverage,
+                'Adults': coverage
+            }
+        }
+
+        # Get the appropriate update based on the target
+        if target in updates:
+            prognosed_mda.update(pd.DataFrame([updates[target]], index=prognosed_mda.index))
+
+        return prognosed_mda
+
+    def _register_symptoms(self) -> None:
         """Register the symptoms with the `SymptomManager`.
         :params symptoms: The symptoms that are used by this module in a dictionary of the form, {<symptom>:
-        <generic_symptom_similar>}. Each symptom is associated with the average healthcare seeking behaviour."""
-        generic_symptoms = self.sim.modules['SymptomManager'].generic_symptoms
-        self.sim.modules['SymptomManager'].register_symptom(*[
-            Symptom(name=_symp) for _symp in symptoms if _symp not in generic_symptoms
-        ])
+        <generic_symptom_similar>}. Each symptom is associated with the average healthcare seeking behaviour
+        unless otherwise specified."""
+
+        # Declare symptoms that this module will cause and which are not included in the generic symptoms:
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='schisto_low',
+                    odds_ratio_health_seeking_in_children=0.01,
+                    odds_ratio_health_seeking_in_adults=0.01)  # no health-seeking
+        )
+
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='ss_sm_moderate'))
+
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='ss_sm_heavy'))
+
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='ss_sh_moderate'))
+
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='ss_sh_heavy'))
```

Comment:
Is it right that these are symptoms - wouldn't the symptom be "itchiness" or something like that, and these be a property of the individual? (Not suggesting we change this at this stage)